doc/classes/ProjectSettings.xml

```diff
@@ -846,7 +846,8 @@
 		</member>
 		<member name="display/window/size/extend_to_title" type="bool" setter="" getter="" default="false">
 			Main window content is expanded to the full size of the window. Unlike a borderless window, the frame is left intact and can be used to resize the window, and the title bar is transparent, but has minimize/maximize/close buttons.
-			[b]Note:[/b] This setting is implemented only on macOS.
+			[b]Note:[/b] This setting is implemented only on macOS, Linux and Windows and when [member display/window/subwindows/embed_subwindows] is [code]false[/code].
```

Comment:
```suggestion
			[b]Note:[/b] This setting is implemented on macOS, Linux, and Windows. It also requires [member display/window/subwindows/embed_subwindows] to be [code]false[/code].
```
But it may also be good to split these in two different notes.
```suggestion
			[b]Note:[/b] This setting is implemented on macOS, Linux, and Windows. 
			[b]Note:[/b] This setting only works if [member display/window/subwindows/embed_subwindows] is [code]false[/code].
```